.devcontainer/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM debian:bullseye
+ENV DEBIAN_FRONTEND=noninteractive
+ENV LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
+RUN apt-get update \
+    && apt-get install -y vim nano git curl gcc ninja-build cmake libudev-dev \
+    python3 python3-pip libusb-1.0-0 libssl-dev pkg-config libtinfo5 clang \
+    && apt-get clean -y && rm -rf /var/lib/apt/lists/* /tmp/library-scripts
+ARG CONTAINER_USER=esp
+ENV USER=${CONTAINER_USER}
+RUN adduser --disabled-password --gecos "" ${CONTAINER_USER}
+USER ${CONTAINER_USER}
+WORKDIR /home/${CONTAINER_USER}
+ARG NIGHTLY_VERSION=nightly-2022-03-10
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- \
+    --default-toolchain ${NIGHTLY_VERSION} -y
+ENV PATH=${PATH}:$HOME/.cargo/bin:$HOME/.cargo/bin
+RUN $HOME/.cargo/bin/rustup component add rust-src --toolchain ${NIGHTLY_VERSION}
+RUN $HOME/.cargo/bin/rustup target add riscv32i-unknown-none-elf
+RUN $HOME/.cargo/bin/cargo install cargo-generate espmonitor bindgen ldproxy
+RUN $HOME/.cargo/bin/cargo install cargo-espflash --version 1.4.1
+ENV ESP_IDF_TOOLS_INSTALL_DIR=global
+ENV ESP_BOARD=esp32c3
+ENV ESP_IDF_VER=v4.4
+ENV ESP_IDF_VERSION=release/v4.4
```

Comment:
To understand the value correctly, this is equivalent to values `branch:release/v4.4` (as specified in all exercises) & `tag:v4.4` according to the [Configuration](https://github.com/esp-rs/esp-idf-sys#configuration) section of the Readme?

At a later point (not part of this PR) we should pick a release tag as value, also in all exercises. For one the current reference is still a branch that in theory allows new commits, while a release tag (e.g. `v4.4`) is fix. Or are these branches used to backport version compatible (security) fixes?

Reply:
> To understand the value correctly, this is equivalent to values branch:release/v4.4 (as specified in all exercises) & tag:v4.4 according to the [Configuration](https://github.com/esp-rs/esp-idf-sys#configuration) section of the Readme?

Yes, is equivalent. 
> Or are these branches used to backport version compatible (security) fixes?

Yes, branches allow backports, so changing to tag `v4.4` might be beneficial as you mentioned.